### PR TITLE
Background upgrade

### LINKED ISSUE #3: mibitiff support
Currently, mibitiffs are not supported.  Not only should support be added, but some changes need to be made to image/data/channel storing so that it generalizes mibitiffs and multiple single-page tiffs (generalize as in 'abstract-away' the differences)

### LINKED ISSUE #21: centralize point loading/handling?
'Cohort loading' is only used for viewing tifs, which is nice, but could definitely be used for centralized point loading. 

This would shift the burden of point management onto the main viewer, which has its pros/cons:

Pros:
 - each plugin doesn't need to independently handle points (big pro)
 - point abstraction is consistent
 - cohort loading doesn't feel as gimmicky

Con:
 - peripheral point-related data (label data, cell tables, etc) should be handled by the main viewer (not necessary but looks clunky otherwise imo)

Going to go ahead and say this should happen, but I'm open to reversing course on this

### PR DESCRIPTION
**Pull request by ackagel**
Centralizes point handling, adds new Background Removal plugin, and adds support for mibitiffs.  Closes #22, #21, #3.

### CODE REVIEW

**Comment by ackagel** [comment]
no need for a super in-depth review, just needs a quick look over to make sure nothing's clearly out of the ordinary

File: amp/tiff_utils.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+import numpy as np
+from tifffile import TiffFile, TiffWriter
+import json
+import datetime
+from itertools import compress
+
+import sys
+
+
+def read_mibitiff(file, channels=None, get_metadata=False):
```
Comment (ngreenwald) [inline amp/tiff_utils.py:11]: The utils would probably be the easiest to write tests for

File: plugins/background_removal.plg/background_removal.py [hunk 2]
```diff
@@ -2,22 +2,17 @@
 # end custom imports - DO NOT MANUALLY EDIT ABOVE
 from PyQt5 import QtWidgets, QtCore, uic
 
-from amp.mplwidget import ImagePlot, Plot
-from amp.point import Point
 from amp.main_viewer import MainViewer
-from amp.resource_path import resource_path
+from amp.mplwidget import ImagePlot
```
Comment (ngreenwald) [inline plugins/background_removal.plg/background_removal.py:6]: Pretty major rewrite here. Not really feasible to catch any potential bugs by eye, gotta just try it out and see what breaks (which I'm guessing you already did during development).

**Review by ngreenwald — APPROVED** [review]
A lot of stuff changed here. No objections to the code you've written, but I don't think code review would be able to catch any potential bugs due to how much changed. 

Worth thinking about when the extra effort for starting to test this is going to be worth it. The main priority is getting a working version of denoising for Erin's cohort once she acquires it. The question is whether that will be aided or hindered by setting up a testing framework. That depends on how much time you're currently spending tracking down weird errors as you're writing code.

**Pull request merged**